libs/ui/layout/src/lib/figure/figure.component.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  Component,
+  OnInit,
+  ChangeDetectionStrategy,
+  Input,
+} from '@angular/core'
+
+@Component({
+  selector: 'gn-ui-figure',
+  templateUrl: './figure.component.html',
+  styleUrls: ['./figure.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class FigureComponent {
+  @Input() icon: string
```

Comment:
I think it would be nice to be able to specify the style of the icon as well, since [Material provides many](https://fonts.google.com/icons) (outlined, filled, rounded...).

Maybe this could be done by specifying `[rounded]people` as the icon name for example? (adding another parameter would maybe tie us too much to the Material Icons lib)

Thus anything specified in brackets would be considered a "variant" and simply ignored if not valid/found